lua/starfall/libs_sv/prop.lua

```diff
@@ -468,9 +468,8 @@ function props_library.createSent(pos, ang, class, frozen, data)
 		local sentparams = sent2[1]
 		if data ~= nil then checkluatype(data, TYPE_TABLE) else data = {} end
 		if data.Model and isstring(data.Model) then
-			data.Model = SF.NormalizePath(data.Model)
-			if not (util.IsValidModel(data.Model) and util.IsValidProp(data.Model)) then SF.Throw("Invalid model", 2) end
-			if ply ~= NULL and gamemode.Call("PlayerSpawnProp", ply, data.Model)==false then SF.Throw("Another hook prevented the model for this SENT", 2) end
```

Comment:
💡 GMod [calls both](https://github.com/Facepunch/garrysmod/blob/ce1b6581e63d225ab6606a43fbf7f303028b6841/garrysmod/gamemodes/sandbox/gamemode/commands.lua#L22) `PlayerSpawnObject` and `PlayerSpawnProp` when spawning props, you might want to keep both there

Reply:
We call PlayerSpawnSENT for SENTs